corehq/apps/case_importer/templates/case_importer/partials/ko_import_status.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+<!--ko if: state === $root.states.SUCCESS-->
+    <div class="text-success">
+        <!--ko if: result.match_count-->
+            <p>
+                {% blocktrans with match_count='result.match_count' %}
+                    <strong data-bind="text: {{ match_count }}"></strong> rows were matched and
+                    updated.
+                {% endblocktrans %}
+            </p>
+        <!--/ko-->
+
+        <!--ko if: result.created_count > 0-->
+            <p>
+                {% blocktrans with created_count='result.created_count' %}
+                    <strong data-bind="text: {{ created_count }}"></strong> rows did
```

Comment:
Yeah the django template evaluates this to

```html
<strong data-bind="text: result.created_count"></strong>
```

so it's still a dynamic binding. The idea is that if I change the structure of `result` somehow and want to edit where we're getting that value from, I can change it outside of the blocktrans block, so that the string doesn't have to be translated again. Agree it's pretty awkward and open to better suggestions.